internal/commands/doc.go

```diff
@@ -33,3 +59,42 @@ func NewCmdDoc() *cobra.Command {
 
 	return cmd
 }
+
+func generateSection(name string, t *template.Template) error {
+	s := schema.GetSchema()
+	filename := "README.md"
+	if name != "main" {
+		filename = strings.ToUpper(name) + ".md"
+		s = schema.GetSchema()[name].Items
+	}
+	f, err := os.Create(filepath.Join(".", "website", "schema", filename))
+	if err != nil {
+		return err
+	}
+
+	err = t.Execute(f, Section{
+		Name:  cases.Title(language.Und).String(name),
+		Items: s,
+	})
+	if err != nil {
+		return err
+	}
+	err = f.Close()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+type Section struct {
+	Name string
+	schema.Items
+}
+
+var sectionTmpl = `# {{.Name}} Section
+| Parameter | Default Value | Required | Description |
+| --- | --- | --- | --- |
+{{range $k, $v := .Items}}| {{$k}} | {{$v.Default}} | {{$v.Required}} | {{$v.Description}} |
```

Comment:
Let's use `yes` if the parameter is required and `no` if it's not (and not true/false).

Reply:
If there is no way to get the default, let's remove this field from the template for now (but keep in the struct)